cmd/provider_cmd_squadcast.go

```diff
@@ -29,6 +29,8 @@ func newCmdSquadcastImporter(options ImportOptions) *cobra.Command {
 	cmd.PersistentFlags().StringVarP(&refreshToken, "refresh-token", "", "", "YOUR_SQUADCAST_REFRESH_TOKEN or env param SQUADCAST_REFRESH_TOKEN")
 	cmd.PersistentFlags().StringVarP(&region, "region", "", "", "eu or us")
 	cmd.PersistentFlags().StringVarP(&teamName, "team-name", "", "", "Squadcast team name")
+	cmd.PersistentFlags().StringVarP(&serviceName, "service-name", "", "", "Squadcast service name")
+	cmd.PersistentFlags().StringVarP(&scheduleName, "schedule-name", "", "", "Squadcast schedule name")
```

Comment:
Automation resources i.e `tagging_rules`, `routing_rules`, `deduplication_rules` and `suppression_rules` their API requires `service_id`, which like `team_id` we're assuming the users won't know however, they will know the service name since it's right on their service menu.
So just like `team_id`, we're using `getServiceByName` API to fetch the service and then extract the `service_id`.
![image](https://user-images.githubusercontent.com/72292326/188502160-22246368-e5e4-4b4a-873e-3f9c76ead34c.png)

Reply:
We need the schedule_name in the `schedule` resource, because when we are creating the resource, we need to pass the name of the schedule
![image](https://user-images.githubusercontent.com/110392771/188555033-79c8eef3-76eb-4f94-a19d-5a82cf6ccfc0.png)
![image](https://user-images.githubusercontent.com/110392771/188555118-8c79259a-76b5-493e-8384-4455c47a323a.png)

Reply:
We just figured out that passing `--schedule-name` flag is not required for the `schedule` resource, so I am removing it

Reply:
sure, thanks